src/consensus/params.h

```diff
@@ -17,23 +17,68 @@ namespace Consensus
 {
 enum DeploymentPos
 {
-    DEPLOYMENT_TESTDUMMY,
-    DEPLOYMENT_CSV, // Deployment of BIP68, BIP112, and BIP113.
+    // bip135 begin
+    // List of deployment bits. Known allocated bits should be described by a
+    // name, even if their deployment logic is not implemented by the client.
+    // (their info is nevertheless useful for awareness and event logging)
+    // When a bit goes back to being unused, it should be renamed to
+    // DEPLOYMENT_UNASSIGNED_BIT_x .
     // NOTE: Also add new deployments to VersionBitsDeploymentInfo in versionbits.cpp
+    DEPLOYMENT_CSV = 0, // bit 0 - deployment of BIP68, BIP112, and BIP113.
+    DEPLOYMENT_SEGWIT, // bit 1 - Segregated Witness (BIP141, BIP143, BIP144)
+    // begin unassigned bits. Rename bits when allocated.
+    DEPLOYMENT_UNASSIGNED_BIT_2,
+    DEPLOYMENT_UNASSIGNED_BIT_3,
+    DEPLOYMENT_UNASSIGNED_BIT_4,
+    DEPLOYMENT_UNASSIGNED_BIT_5,
+    DEPLOYMENT_UNASSIGNED_BIT_6,
+    DEPLOYMENT_UNASSIGNED_BIT_7,
+    DEPLOYMENT_UNASSIGNED_BIT_8,
+    DEPLOYMENT_UNASSIGNED_BIT_9,
+    DEPLOYMENT_UNASSIGNED_BIT_10,
+    DEPLOYMENT_UNASSIGNED_BIT_11,
+    DEPLOYMENT_UNASSIGNED_BIT_12,
+    DEPLOYMENT_UNASSIGNED_BIT_13,
+    DEPLOYMENT_UNASSIGNED_BIT_14,
+    DEPLOYMENT_UNASSIGNED_BIT_15,
+    DEPLOYMENT_UNASSIGNED_BIT_16,
+    DEPLOYMENT_UNASSIGNED_BIT_17,
+    DEPLOYMENT_UNASSIGNED_BIT_18,
+    DEPLOYMENT_UNASSIGNED_BIT_19,
+    DEPLOYMENT_UNASSIGNED_BIT_20,
+    DEPLOYMENT_UNASSIGNED_BIT_21,
+    DEPLOYMENT_UNASSIGNED_BIT_22,
+    DEPLOYMENT_UNASSIGNED_BIT_23,
+    DEPLOYMENT_UNASSIGNED_BIT_24,
+    DEPLOYMENT_UNASSIGNED_BIT_25,
+    DEPLOYMENT_UNASSIGNED_BIT_26,
+    DEPLOYMENT_UNASSIGNED_BIT_27,
+    DEPLOYMENT_TESTDUMMY, // bit 28 - used for deployment testing purposes
+    // bip135 end
     MAX_VERSION_BITS_DEPLOYMENTS
```

Comment:
Also, what is the relationship between this and VERSIONBITS_NUM_BITS?  If they are the same, please consolidate. 1 idea should have exactly 1 constant to help code maintenance.  Otherwise you could have a case where only 1 of the 2 constants gets changed.